script/rhcos_versions.sh

```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/env bash
 
 set -x
-set -e
```

Comment:
Leave it, just mask the failure:
```
RHCOSVERSION="$(oc adm release info "${VERSION}" -o 'jsonpath={.displayVersions.machine-os.Version}') 2>&1" ||
  printf 'Continue with the error %s.\n' "$RHCOSVERSION"
```

Reply:
Do you mean the script exits anyway?

Reply:
No I copied and pasted your suggestion into my code and it doesn't run the way it should.

Reply:
Try now, show me the output.

Reply:
Edits committed.  Thanks!